dts/msm8916/msm8916-huawei-hx4qcom.dts

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+#include <skeleton.dtsi>
+
+/ {
+	qcom,msm-id = <206 0>,<248 0>,<249 0>,<250 0>;
+	qcom,board-id = <8034 4>,<8050 4>,<8082 4>,<8098 4>,<8114 4>,<8115 4>,<8130 4>,<8131 4>;
+
+	model = "Huawei Honor 4X Qualcomm";
+	compatible = "huawei,h4xqcom", "qcom,msm8916", "lk2nd,device";
+
+	panel {
+		compatible = "huawei,h4xqcom";
```

Comment:
```suggestion
		compatible = "huawei,h4xqcom-panel";
```

or

```suggestion
		compatible = "huawei,cherry-panel";
```

Reply:
many Honor 4X Kirin codename is cherry too,so i think use new codename is better to use "cherry"

Reply:
btw,this is my first pull request(maybe),i will do what to continue pull request